servicemesh-install/topics/servicemesh-istio-comparison.adoc

```diff
@@ -45,3 +47,57 @@ spec:
     properties:
       request.regex.headers[<header>]: "<regular expression>"
 ----
+
+== Automatic route creation
+{ProductName} automatically manages OpenShift routes for Istio gateways. When an Istio gateway is created, updated, or deleted in the {ProductShortName}, a matching OpenShift route is created, updated, or deleted.
+
+If the following gateway is created:
+
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: gateway1
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - www.bookinfo.com
+    - bookinfo.example.com
+----
+
+The following OpenShift routes are automatically created:
+
+----
+$ oc -n istio-system get routes
+NAME              HOST/PORT                                            PATH      SERVICES               PORT      TERMINATION   WILDCARD
+gateway1-lvlfn    bookinfo.example.com                                           istio-ingressgateway   <all>                   None
+gateway1-scqhv    www.bookinfo.com                                               istio-ingressgateway   <all>                   None
+----
+
+If this gateway is deleted, {ProductName} will delete the routes.
+
+[NOTE]
+====
+Manually created routes are not managed by the {ProductShortName}.
+====
+
+=== Catch-all domains
+Catch-all domains are not supported. If a catch-all domain is found in the gateway definition, {ProductName} will create the route but https://docs.okd.io/latest/architecture/networking/routes.html#route-hostnames[relies on OpenShift to create a default hostname]. The route will *not* be a catch-all route and will have a hostname with a `<route-name>[-<namespace>].<suffix>` structure.
+
+=== Subdomains
+Subdomains are supported, but they are not enabled by default in OpenShift. {ProductName} will create the route with the subdomain, but it will only work after you enable subdomains in OpenShift. See the https://docs.okd.io/latest/install_config/router/default_haproxy_router.html#using-wildcard-routes[OpenShift documentation on Wildcard Routes] for more information.
+
+=== TLS
+OpenShift routes are configured to support TLS.
```

Comment:
by default?  Or upstream just doesn't' support TLS at all?

Reply:
If the Gateway contains a `tls` section, the OpenShift Route will be configured to support TLS.